src/widgets/peoplePicker.js

```diff
@@ -28,6 +28,7 @@ export class PeoplePicker {
     this.groupPickedCb = groupPickedCb
     this.selectedgroup = this.options.selectedgroup // current selected group if any
     this.onSelectGroup = this.onSelectGroup.bind(this)
+    this.findAddressBook = this.findAddressBook.bind(this)
```

Comment:
You don't need to do this if it's already moved into the class. This happens implicitly.